semgrep/rules/python/xss/reflected_xss.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+  - id: reflected-cross-site-scripting
+    patterns:
+      - pattern-not-inside: |
+          import phantom
+          ...
+          class $CLASS(phantom.BaseConnector):
+            ...
+      - pattern-either:
+        - pattern-inside: |
+            def $FUNC($REQUEST, ...):
+              ...
+        - pattern-inside: |
+            def $FUNC(self, ...):
+              ...
+      - pattern: |
+          return django.http.HttpResponse($C, ...)
+      - pattern-not: |
+         return django.http.HttpResponse($C, ..., content_type='text/plain', ...)
```

Comment:
I still find it really cool how this can automagically deal with arbitrary parameters and imports from such a simple syntax 😁

Reply:
yep :) it also handles variations with single/double quotes and spaces in keyword declarations